dts/arm/nordic/nrf52832.dtsi

```diff
@@ -46,6 +46,24 @@
 			label = "UART_0";
 		};
 
+		gpiote0: gpiote@40006000 {
+			compatible = "nordic,nrf-gpiote";
```

Comment:
nordic,nrf**5**-gpiote

Reply:
Thanks! I can't believe how many times I looked over that. Quite embarrassing :|

Reply:
Frequent issue :-) you're not the only one (count me in!)